code/__DEFINES/construction.dm

```diff
@@ -92,17 +92,19 @@
 #define CAT_BREAD	"Breads"
 #define CAT_BURGER	"Burgers"
 #define CAT_CAKE	"Cakes"
+#define CAT_DONUT	"Donuts"
+#define CAT_EASTERN	"Eastern & Fish"
```

Comment:
I told you not to add eastern food to this category. Leave them where you found em.